runtime/dolphin/src/lib.rs

```diff
@@ -552,6 +553,7 @@ impl pallet_treasury::Config for Runtime {
     type MaxApprovals = ConstU32<100>;
     type WeightInfo = weights::pallet_treasury::SubstrateWeight<Runtime>;
     type SpendFunds = ();
+    type SpendOrigin = frame_support::traits::NeverEnsureOrigin<Balance>;
```

Comment:
this is weird. There's an EnsureNever trait that should be doing the same thing, no need to implement one.
https://snowbridge-rust-docs.snowfork.com/frame_system/struct.ensurenever?search=

Reply:
`SpendOrigin` expects an implementation of `EnsureOrigin` with a `Success` generic which is the the maximum amount that this origin is allowed to spend at a time.
Let me add this as a comment.